Lamdu/Expr/Val/Arbitrary.hs

```diff
@@ -76,8 +76,7 @@ arbitraryLeaf :: GenExpr V.Leaf
 arbitraryLeaf = do
     Env locals globals <- Reader.ask
     join . liftGen . Gen.elements $
-        [ V.LLiteralInteger <$> liftGen arbitrary
```

Comment:
This is a slight difference from past behavior, though probly not too important currently..